docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2956,13 +2956,14 @@ added later (not in the minimum viable model), if desired.
 
   * - Unique simulant ID (for PRL tracking)
   * - First name
-  * - Middle initial
+  * - Middle name
   * - Last name
   * - DOB (stored as a string in YYYYMMDD format, as indicated by [CARRA_SSA]_ Table 1)
   * - Sex (binary; "Male" or "Female")
   * - Social Security Number
   * - Type of event
   * - Date of event (stored as a string in YYYYMMDD format, as indicated by [CARRA_SSA]_ Table 1)
+  * - Event ID (unique integer identifier for each row in the SSA dataset, representing a ground-truth identifier for the event recorded in that row; unaffected by noise functions; to be used for comparing noised and unnoised data)
```

Comment:
We probably don't need to require that this be an integer; I know for e.g. simulant ID we have an underscore included to make it easier to do this across shards.

Reply:
Well, I **want** these all to be changed to integers because strings make the data bloated and slow everything down, so I figured I'd pre-empt the problem by specifying what I want in the concept model. It's just as easy to have a shard prefix with integers as with strings.

Reply:
Fair enough! I think this does mean that the engineers will need to hard-code a maximum number of of simulants per shard in order to "zero-pad" the simulant ID part (otherwise shard 12, simulant 84 and shard 1, simulant 284 would have the same simulant_id). Let's see what their reaction to this is.

Reply:
In this case, it would be the event ID not the simulant ID that gets zero-padded, which is even harder to hard-code a maximum for. Maybe they'd flip it and zero-pad the shard in the low-order digits. Or maybe I'm missing some obviously better solution?

Reply:
With 64-bit ints we have a lot of wiggle room. First note that 2**64 > 1.8e19. Suppose we set unreasonably high maximums of 100 million simulants per shard and 10,000 SSA events per simulant. That's a maximum of 1e8 * 1e4 = 1e12 events for each shard of SSA data (that's 1 trillion SSA events per shard!). That still leaves room for more than 1.8e19 / 1e12 = 1.8e7 shards (i.e., over 18 million shards), which should be way more than sufficient. So I think it shouldn't be too hard to set reasonable pad lengths to have well-defined event IDs across shards, regardless of whether we put shards or events in the low-order digits.